WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -208,6 +226,46 @@ class ReaderDetailWebviewViewController: UIViewController, ReaderDetailView {
         coordinator?.share(fromView: sender)
     }
 
+    /// Start following the scroll view to hide nav bar and toolbar
+    /// Only if VoiceOver is not active
+    ///
+    private func followScrollView() {
+        if isFollowingScrollView,
+            UIAccessibility.isVoiceOverRunning {
+            return
+        }
+
+        if let navigationController = navigationController as? ScrollingNavigationController {
+            navigationController.followScrollView(scrollView, delay: 50.0, followers: [NavigationBarFollower(view: toolbarContainerView, direction: .scrollDown)])
```

Comment:
Maybe move the delay to a constant?

Reply:
Done in f1e5981a63ff162a065197703de3e41016a008c7